webpack.dev.js

```diff
@@ -46,13 +41,15 @@ module.exports = {
       },
       {
         test: /\.css$/,
-        loader: "style!css",
+        use: [
+          {loader:"style-loader"}, {loader: "css-loader"}
+        ]
       },
-      { test: /\.png$/, loader: "url?limit=10000" },
-      { test: /\.eot(\?v=\d+\.\d+\.\d+)?$/, loader: "file" },
-      { test: /\.(woff|woff2)$/, loader:"url?prefix=font/&limit=5000" },
-      { test: /\.ttf(\?v=\d+\.\d+\.\d+)?$/, loader: "url?limit=10000&mimetype=application/octet-stream" },
-      { test: /\.svg(\?v=\d+\.\d+\.\d+)?$/, loader: "url?limit=10000&mimetype=image/svg+xml" }
+      { test: /\.png$/, loader: "url-loader", options: {"limit": 10000} },
+      { test: /\.eot(\?v=\d+\.\d+\.\d+)?$/, loader: "file-loader" },
+      { test: /\.(woff|woff2)$/, loader:"url-loader", options: {"prefix": "font/", "limit": 5000} },
+      { test: /\.ttf(\?v=\d+\.\d+\.\d+)?$/, loader: "url-loader", options: {"limit": 10000, "mimetype": "application/octet-stream"} },
+      { test: /\.svg(\?v=\d+\.\d+\.\d+)?$/, loader: "url-loader", options: {"limit": 10000, "mimetype": "image/svg+xml"} }
```

Comment:
Nit: no need to wrap the `limit` and `mimetype` property names around double quotes.